modules/accounting/includes/functions/products.php

```diff
@@ -64,7 +64,8 @@ function erp_acct_get_all_products( $args = [] ) {
             WHERE product.product_type_id<>3";
 
         if ( ! empty( $args['s'] ) ) {
-            $sql .= " AND product.name LIKE '%{$args['s']}%'";
+            $sql .= " AND product.name LIKE %s";
+            $sql = $wpdb->prepare($sql, "%". $args['s'] ."%" );
```

Comment:
`$wpdb` have a `esc_like()` method. We can use it here. Also 1 space is expected after opening parenthesis. So the line could be:
`$sql = $wpdb->prepare( $sql, '%' . $wpdb->esc_like( $args['s'] ) . '%' );`